test/k8sT/Services.go

```diff
@@ -1124,6 +1130,20 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`, helpers.DualStackSupp
 			}
 
 			if helpers.RunsOnGKE() {
+				if bpfNodePort && helpers.RunsWithoutKubeProxy() {
```

Comment:
Let's remove the `bpfNodePort && helpers.RunsWithoutKubeProxy()` condition. Otherwise it won't be tested on the current GKE CI job (see https://github.com/cilium/cilium/pull/14793#issuecomment-772379577).

Reply:
Fixed.